src/test/java/seedu/address/logic/parser/ClientNoteAddCommandTest.java

```diff
@@ -3,22 +3,26 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.fail;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NOTE;
 
 import org.junit.jupiter.api.Test;
 
 import seedu.address.commons.core.index.Index;
 import seedu.address.logic.commands.ClientNoteAddCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 import seedu.address.model.note.Note;
+import seedu.address.model.tag.Tag;
 
 
 public class ClientNoteAddCommandTest {
 
+    private static final String SPACE = " ";
```

Comment:
Is this needed lol

Reply:
Pls let me leave this here 😢 . IMO it improves readability in the user input variables later in the tests.